src/App.tsx

```diff
@@ -1,11 +1,13 @@
 import { Outlet } from 'react-router-dom';
 import Header from './components/Header';
+import Footer from './components/Footer';
 
 function App() {
    return (
       <div>
          <Header />
          <Outlet />
+         <Footer />
       </div>
    );
 }
```

Comment:
I don't know how it got to main with the router not working on Codux, but here it seems even worse, so fix it and then I will approve, because this merge main to here didn't work well.